app/app.js

```diff
@@ -62,10 +62,37 @@ export default class App {
         // Usage deeplinking
         Linking.addEventListener('url', this.handleDeepLink);
 
+        this.setFontFamily();
         this.getStartupThemes();
         this.getAppCredentials();
     }
 
+    setFontFamily = () => {
+        // Set a global font for Android
+        if (Platform.OS === 'android') {
```

Comment:
Is it possible to do this from Android's styles.xml?

https://stackoverflow.com/questions/16404820/how-to-set-default-font-family-for-entire-android-app

Reply:
I was following this https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e and somehow it gives me more confidence that is going to use the right font